moderators.md

```diff
@@ -0,0 +1,143 @@
+and is governed by the official [R-multiverse review policy](review.md).
+
+The role of a [moderator](governance.md#moderator) is to read contributions at <https://github.com/r-multiverse/contributions/pulls>,
+merge the pull requests that comply with [R-multiverse official policies](policies.md),
+and either close the pull request (without merging) or ask for changes in non-compliant cases.
+
+As a [moderator](governance.md#moderator), you may optionally restrict your review to packages in your area of expertise.
+Simply search [contributions](https://github.com/r-multiverse/contributions/pulls) for relevant keywords,
+including [R-multiverse topics](https://r-multiverse.org/topics/).
+
+## Topics
+
+We encourage moderators to contribute [R-multiverse topics](https://r-multiverse.org/topics/)
+about their fields of expertise.
+To do so, submit a pull request to <https://github.com/r-multiverse/topics>^[
+Through triage access,
+R-multiverse [moderators](governance.md#moderator) are authorized to merge these pull requests.
+Please only edit the plain files without file extensions.
+All the HTML files are automatically generated.
+]
+to add a text file describing the subject matter ([example here](https://github.com/r-multiverse/topics/blob/main/bayesian)).
+A bot periodically rebuilds the website based on these text files.
+Seasoned [contributors](governance.md#contributor) will see your topic [listed here](https://r-multiverse.org/topics/)
+and add keywords to their [contributions](https://github.com/r-multiverse/contributions/pulls) accordingly.
+
+
+## Discussions
+
+[Moderators](governance.md#moderator) help answer questions on the [`r-multiverse/help`](https://github.com/r-multiverse/help)
+[issue](https://github.com/r-multiverse/help/issues) and [discussion](https://github.com/r-multiverse/help/discussions) forums.
+Questions may ask about specific packages, package reviews, the workings of R-multiverse, etc.
+A [moderator](governance.md#moderator) may close an
+[issue](https://github.com/r-multiverse/help/issues) or [discussion](https://github.com/r-multiverse/help/discussions) thread
+if it originally came from a [contributor](governance.md#contributor) or [user](governance.md#user)
+and the original issue is resolved.
+[Moderators](governance.md#moderator) should not close issues or discussions created by [administrators](team.md#administrators),
+including [official proposals](governance.md#proposals) to modify [R-multiverse policies](policies.md).
+
+## Enforcement
+
+[Moderators](governance.md#moderator) must understand [all R-multiverse policies](policies.md)
+and be prepared to enforce them.
+[Moderators](governance.md#moderator) must respond immediately and quickly to the following rare cases:
+
+1. If an R-multiverse package or [R-multiverse stakeholder](governance.md#r-multiverse-stakeholders) violates any applicable laws,
+contact the appropriate law enforcement agency.
```

Comment:
Why not contact the administrators team. That seems like a tough thing to do and I wouldn't expect an _individual_ to go through it alone.

Reply:
Updated in 226eb8164f691125613c132b5fd8b5c65c3662e4